build.yaml

```diff
@@ -1031,6 +1048,48 @@ job_type:
       clean_repo: true
       archive_artifacts: *acceptance_tests_artifacts_ubuntu_special_case
       timeout: 45
+    run_acceptance_loopback_on_AWS_CentOS_7_for:
+      on_nodes_with_labels: 'aws-centos-7-SELinux-T2Medium'
+      with_modules:
+        - flocker.acceptance
```

Comment:
:grey_question: Slight preference for extracting this into some variable "run_acceptance_modules_lumped" and/or adding a comment. Probably something to the effect of:

"For setups that run quickly enough we run all of the acceptance tests together rather than sharding them out across projects in Jenkins."

If that's _not_ the reason you lumped them together here then definitely add a comment.